include/solvers/time_solver.h

```diff
@@ -267,6 +278,18 @@ class TimeSolver : public ReferenceCountedObject<TimeSolver>
    */
   virtual Real last_complete_deltat();
 
+  /**
+   * Setter and getter for the final time variable
```

Comment:
Needs more documentation, too.  At first glance it just raises a ton of "Do I have to set this?", "What's it get used for?", "What if I want to restart a simulation to an extended time?" sorts of questions.

Reply:
Fix commit: